cmd/resource-topology-exporter/main.go

```diff
@@ -112,7 +112,8 @@ func parseArgs(args ...string) (ProgArgs, error) {
 	refCnt := flags.String("reference-container", "", "Reference container, used to learn about the shared cpu pool\n See: https://github.com/kubernetes/kubernetes/issues/102190\n format of spec is namespace/podname/containername.\n Alternatively, you can use the env vars REFERENCE_NAMESPACE, REFERENCE_POD_NAME, REFERENCE_CONTAINER_NAME.")
 
 	flags.StringVar(&pArgs.RTE.NotifyFilePath, "notify-file", "", "Notification file path.")
-	flags.Int64Var(&pArgs.RTE.MaxEventsPerSecond, "max-events-per-second", 1, "Max times per second resources will be scanned and updated")
+	flags.Int64Var(&pArgs.RTE.MaxEventsPerTimeUnit, "max-events-per-time-unit", 1, "Max times per time unit that resources will be scanned and updated.\n See 'time-unit-to-limit-events' to set the base time unit")
+	flags.DurationVar(&pArgs.RTE.TimeUnitToLimitEvents, "time-unit-to-limit-events", 100*time.Millisecond, "Base time unit to calculate the maximum rate of events. 'max-events-per-time-unit' / 'time-unit-to-limit-events")
```

Comment:
Not a big fan of the chosen name but I didn't found anything better.
Default value just as an example. 
Suggestions would be really appreciated for both name and default value.

Reply:
This is going to be hard to tune, so I'd just expose a `-max-events-per-second` flag and compute the values internally.
IOW, let's start with the simplest user-facing interface. We can keep the current internal interfaces.